src/Compilers/Core/Portable/Symbols/IFunctionPointerTypeSymbol.cs

```diff
@@ -5,10 +5,20 @@
 
 namespace Microsoft.CodeAnalysis
 {
-    // PROTOTYPE(func-ptr): Document
+    /// <summary>
+    /// Represents a function pointer type such as "delegate*&lt;void&gt;". Pointer types
+    /// are used only in unsafe code.
```

Comment:
Is this statement necessary (or correct)? Won't we expose an `IFunctionPointerTypeSymbol` for uses in code not marked `unsafe`?

Reply:
It's copied from IPointerTypeSymbol. I can remove it though.

---
In reply to: [431341498](https://github.com/dotnet/roslyn/pull/44436#discussion_r431341498) [](ancestors = 431341498)